### PR TITLE
Calico/Vpp docs cleanup

### PR DESCRIPTION
**Pull request by onong**
Cleaning up the outdated content from the docs and fixing some typos.

### CODE REVIEW

**Review by sknat — DISMISSED** [review]
lgtm !
Thanks @onong

**Review by lwr20 — COMMENTED** [review]
LGTM

File: calico/getting-started/kubernetes/vpp/getting-started.mdx [hunk 1]
```diff
@@ -378,9 +378,9 @@ The supported values will depend on the interface type. Available values are:
 
 **Legacy options**
 
-We maintain legacy support for the `CALICOVPP_INTERFACE` env var and `CALICOVPP_NATIVE_DRIVER` that corresponds to the following:
+We maintain legacy support for the `CALICOVPP_INTERFACE` and `CALICOVPP_NATIVE_DRIVER` env vars that corresponds to the following:
 
-`CALICOVPP_INTERFACES` -> `uplinkInterfaces[0].interfaceName`
+`CALICOVPP_INTERFACE` -> `uplinkInterfaces[0].interfaceName`
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/vpp/getting-started.mdx:383]: Nice catch! Thanks!

File: calico/getting-started/kubernetes/vpp/getting-started.mdx [hunk 2]
```diff
@@ -378,9 +378,9 @@ The supported values will depend on the interface type. Available values are:
 
 **Legacy options**
 
-We maintain legacy support for the `CALICOVPP_INTERFACE` env var and `CALICOVPP_NATIVE_DRIVER` that corresponds to the following:
+We maintain legacy support for the `CALICOVPP_INTERFACE` and `CALICOVPP_NATIVE_DRIVER` env vars that corresponds to the following:
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/vpp/getting-started.mdx]:
```suggestion
We maintain legacy support for the `CALICOVPP_INTERFACE` and `CALICOVPP_NATIVE_DRIVER` environment variables:
```

File: calico/reference/vpp/uplink-configuration.mdx [hunk 3]
```diff
@@ -15,52 +15,54 @@ You can choose different ways to consume the host's primary interface with VPP,
 
 ## General mechanics
 
-The main interface configuration is controlled by a variable named `CALICOVPP_NATIVE_DRIVER` that is passed to the `vpp` container.
-You can edit the .yaml file as follows or use the overlays provided in `./yaml/overlays/*`.
+The `calico-vpp-config` ConfigMap section of the manifest yaml contains the key `CALICOVPP_INTERFACES` which is a dictionary with parameters
```
Comment (ctauchen) [inline calico/reference/vpp/uplink-configuration.mdx]:
```suggestion
The `calico-vpp-config` ConfigMap section of the manifest file contains the key `CALICOVPP_INTERFACES`, which is a dictionary with parameters
```

File: calico/reference/vpp/uplink-configuration.mdx [hunk 4]
```diff
@@ -80,32 +82,11 @@ Ensure your kernel is at least `5.4` with `uname -r`
 
 :::
 
-- Pass `CALICOVPP_NATIVE_DRIVER = af_xdp` to the `vpp` container
+- For the primary interface, ie, `uplinkInterfaces[0]`, set `vppDriver` as "af_xdp"
```
Comment (ctauchen) [inline calico/reference/vpp/uplink-configuration.mdx]:
```suggestion
- For the primary interface, `uplinkInterfaces[0]`, set `vppDriver` to "af_xdp"
```

File: calico/reference/vpp/uplink-configuration.mdx [hunk 5]
```diff
@@ -80,32 +82,11 @@ Ensure your kernel is at least `5.4` with `uname -r`
...
-          env:
-            - name: CALICOVPP_NATIVE_DRIVER
-              value: 'af_xdp'
-            - name: CALICOVPP_RING_SIZE
-              value: '1024'
-            - name: CALICOVPP_RX_QUEUES
-              value: '1'
-            - name: CALICOVPP_RX_MODE
-              value: 'polling'
 ---
```
Comment (ctauchen) [inline calico/reference/vpp/uplink-configuration.mdx:91]: Should this line be removed too?

**Review by ctauchen — COMMENTED** [review]
A few small suggestions. I've made comments only on version 'next', but they apply equally to the same files in version-3.25.

File: calico/getting-started/kubernetes/vpp/getting-started.mdx (hunk 1 above)
Comment (onong) [reply calico/getting-started/kubernetes/vpp/getting-started.mdx:383]: Thanks!

File: calico/getting-started/kubernetes/vpp/getting-started.mdx (hunk 2 above)
Comment (onong) [reply calico/getting-started/kubernetes/vpp/getting-started.mdx]: Done. Thanks for catching it.

File: calico/reference/vpp/uplink-configuration.mdx (hunk 3 above)
Comment (onong) [reply calico/reference/vpp/uplink-configuration.mdx]: Done. Thanks for catching it.

File: calico/reference/vpp/uplink-configuration.mdx (hunk 4 above)
Comment (onong) [reply calico/reference/vpp/uplink-configuration.mdx]: Done for all the occurrences of similar lines for the other drivers too. Thanks!

File: calico/reference/vpp/uplink-configuration.mdx (hunk 5 above)
Comment (onong) [reply calico/reference/vpp/uplink-configuration.mdx:91]: This already removed I think.

**Pull request merged**